chartmogul/api/activity.py

```diff
@@ -7,23 +7,28 @@ class Activity(Resource):
     """
     https://dev.chartmogul.com/v1.0/reference#list-customer-subscriptions
     """
-    _path = "/customers{/uuid}/activities"
+    _path = "/activities"
     _root_key = 'entries'
-    _many = namedtuple('Activities', [_root_key, "has_more", "per_page", "page"])
+    _many = namedtuple('Activities', [_root_key, "has_more", "per_page"])
 
     class _Schema(Schema):
-        id = fields.Int()
         activity_arr = fields.Number(data_key='activity-arr')
         activity_mrr = fields.Number(data_key='activity-mrr')
         activity_mrr_movement = fields.Number(data_key='activity-mrr-movement')
         currency = fields.String()
-        currency_sign = fields.String(data_key='currency-sign')
         date = fields.DateTime()
         description = fields.String()
         type = fields.String()
+        subscription_external_id = fields.String(data_key='subscription-external-id')
+        plan_external_id = fields.String(data_key='plan-external-id')
+        customer_name = fields.String(data_key='customer-name')
+        customer_uuid = fields.String(data_key='customer-uuid')
+        customer_external_id = fields.String(data_key='customer-external-id')
+        billing_connector_uuid = fields.String(data_key='billing-connector-uuid')
+        uuid = fields.String(data_key='uuid')
 
         @post_load
         def make(self, data, **kwargs):
             return Activity(**data)
 
-    _schema = _Schema()
+    _schema = _Schema(unknown=EXCLUDE)
```

Comment:
Added this unknown=EXCLUDE to allow for serving the object even when extra unknown API fields are present. Only known ones will be serialised.
There is an include option as well if that is preferable - https://marshmallow.readthedocs.io/en/stable/quickstart.html#handling-unknown-fields

But we need one of these as without it, it just raises and error when API is even slightly different that what is expected. Customer scoped Activity end point was broken due to this.